plugins/operators/model_evaluation/__init__.py

```
@@ -0,0 +1,1052 @@
+        # NOTE: saved views goes through this
+        if isinstance(values, list):
+            values = {v: 0 for v in values}
+
+        for label, count in values.items():
+            formatted_count = f"{count:,}"
+            formatted_label = f"{label}" + (
+                f" - {formatted_count}" if count > 0 else ""
+            )
+            obj.bool(
+                label,
+                default=True if label in selected_values else False,
```

Ruff (0.8.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)